src/js/popup.js

```diff
@@ -1,4 +1,4 @@
-$(function() {
+$(function () {
```

Comment:
Was this space automatically inserted by VSCode?

Just wondering.

Reply:
Yes I believe so, I use Prettier extension for my VSCode. It was most likely auto-formatting.